tools/msmetaenhancer/msmetaenhancer_wrapper.py

```diff
@@ -16,6 +16,8 @@ def main(argv):
 
     app = Application(log_file=args.log_file)
 
+    # set matchms logging level to avoid extensive messages in stdout
+    set_matchms_logger_level("ERROR")
     # import .msp file
     app.load_spectra(args.input_file, file_format='msp')
 
```

Comment:
The level can be returned back to `WARNING` after loading the spectra.

Reply:
Good point, I set it back to `WARNING` before curation in [d3e8037](https://github.com/RECETOX/galaxytools/pull/259/commits/d3e8037f9e30a2fe2d25c6dafb890810a4aca0bb).